src/FormModel.php

```diff
@@ -26,17 +26,14 @@
 abstract class FormModel implements FormModelInterface, PostValidationHookInterface, RulesProviderInterface
 {
     private array $attributes;
-    /** @psalm-var array<string, array<array-key, string>> */
-    private array $attributesErrors = [];
-    private string $formErrorsClass = FormErrors::class;
     private FormErrorsInterface $formErrors;
     private ?Inflector $inflector = null;
     private bool $validated = false;
 
-    public function __construct()
+    public function __construct(string $formErrorsClass = FormErrors::class)
```

Comment:
I'll need to accept/pass this argument in every form model I create. Is it possible to pass it with a method later?

Reply:
fixed.